project.json

```diff
@@ -75,6 +75,21 @@
           "target": "build"
         }
       ]
+    },
+    "run-history": {
```

Comment:
Please use **all** dependencies from the other commands, including the `plugin-js-packages`.

Reply:
done